frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "bootstrap": "^4.5.3",
     "bootstrap-social": "^5.1.1",
     "font-awesome": "^4.7.0",
+    "joi-browser": "^13.4.0",
```

Comment:
The netlify build is failing by throwing `4:37:52 PM: Browserslist: caniuse-lite is outdated. Please run the following command: `npx browserslist --update-db`
4:39:00 PM: Failed to compile.
4:39:00 PM: 
4:39:00 PM: ./src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx
4:39:00 PM: TypeError: [BABEL] /opt/build/repo/frontend/src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx: api.assumption is not a function (While processing: "/opt/build/repo/frontend/node_modules/babel-preset-react-app/index.js$0$2")` this err.or. Probably the `joi-browser` version is incompatible with `browserslist` can you please try with a different version maybe?

Reply:
I guess the  build issue was because I somehow missed to push the changes of package-lock.json. I have added them now

Reply:
Hey the PR should never contain the changes of `package-lock.json` file in general.